README.rst

```diff
@@ -292,25 +292,26 @@ Configuration File to Pass Options
 
 Beside command line, you can pass options through the configuration file.
 Configuration file is in JSON or YAML_ format with a simple structure.
+The values for 'min' and 'max' can be set with environment variables.
```

Comment:
```suggestion
The values for 'min' and 'max' can be set with environment variables as
`MIN_SPHINX_WARNINGS` and `MAX_SPHINX_WARNINGS`.
```

Reply:
```suggestion
The values for 'min' and 'max' can be set with environment variables,
e.g. `$MAX_SPHINX_WARNINGS`. 
```
I think this would be clearer.

Reply:
Like this? db5976863915bb5324cdcd210949b476cff08de4

Reply:
Cool.